### ISSUE TITLE
[feat]: Add "Setup Backups" as an option during Initial Setup

### ISSUE THREAD

**Issue by kn0wmad** [Start9Labs/embassy-os#1858]
### Prerequisites

- [X] I have searched for [existing issues](https://github.com/start9labs/embassy-os/issues) that already suggest this feature.

### Describe the Feature Request

Backups are of utmost importance, and are often punted on.  In this case, setting up of backups can be front and center, and if opted out of, there is an opportunity for messaging regarding what they are skipping out on, and how nobody can save their shit if they choose to never do a backup.

### Describe the Use Case

self-explanatory

### Describe Preferred Solution

We may want #1848 first

**Comment by BitcoinMechanic** [comment]
I love it as an option, maybe it's too much headache before external drive plugged into the Embassy becomes the standard rather than CIFS (assuming that is the plan).

**Comment by kn0wmad** [comment]
It may be a serious lift on the setup flow side, but we'll see.  I think it's important.  Maybe instead of during setup, it's offered at first boot into the UI.   Needs concept ACK

**Comment by BitcoinMechanic** [comment]
It's basically "the right thing to do" because we should be encouraging backups, but I imagine there'll be a lot of people unable to get CIFS working that then have a really frustrating initial setup.

**Comment by dr-bonez** [comment]
I'd be more a fan of constant nagging in the ui if it's not setup. like a banner. But @MattDHill should weigh in

**Comment by k0gen** [comment]
Agreed. There should be an extra backup step (that user can skip) during the initial setup. Also worth adding something similar when doing OTA system updates (like an alert popup or something like that).

**Comment by kn0wmad** [comment]
#1441 is worth a mention in regard to helping users 'stay safe' - not sure what that lift looks like.
